packages/dashboard/src/components/Wizard/SetupWizardActionBar.styles.ts

```diff
@@ -1,5 +1,9 @@
 import { ISetupWizardActionBarStyles } from './SetupWizardActionBar.types';
 
+const actionLinkStyle = {
+  color: '#605E5C'
```

Comment:
Use the Fluent NeutralColors palette variables here. 
https://fluentfabric.azurewebsites.net/#/color/neutrals